### DISCUSSION TITLE
Apple news中audio无法听取

### DISCUSSION THREAD

**Discussion by blackfiry** [NSRingo/.github#30] [Q&A 问答] [answered]
如题，其他按照操作一步步下来没什么问题。自己尝试了下，把shadowrocket中全局代理改为全部代理后，audio就可以听取。盲猜是没有把audio的域名添加代理规则。

**Comment by VirgilClyne** [comment, marked as answer]
[📰 News](https://nsringo.github.io/guide/apple-news) 早已不再支持 `Shadowrocket`
`Shadowrocket` 也不支持 `URL-REGEX` 类型分流规则
